src/main/java/uk/gov/hmcts/cmc/claimstore/events/solicitor/RepresentedClaimIssuedEvent.java

```diff
@@ -4,19 +4,22 @@
 import uk.gov.hmcts.cmc.domain.models.Claim;
 
 import java.util.Objects;
+import java.util.Optional;
 
 import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
 
 public class RepresentedClaimIssuedEvent {
 
     private final Claim claim;
-    private final String representativeName;
+    private final Optional<String> representativeName;
     private final String representativeEmail;
+    private final String authorisation;
 
-    public RepresentedClaimIssuedEvent(Claim claim, final String submitterName) {
+    public RepresentedClaimIssuedEvent(Claim claim, final Optional<String> submitterName, final String authorisation) {
```

Comment:
break into multiple lines please and don't use Optional as a constructor parameter